CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog for @pdc/service
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
What does this mean? Is it a placeholder for future changes? If empty, perhaps leave it out.

Reply:
This is something of a permanent placeholder, but I'll take it out for now since we need to figure out how and when we tack on a full blown version.

I would normally only update versions *ON RELEASE* not simply on merge.  This would mean that a PR like this would *not* update a version, but would just document changes in `[Unreleased]`.

When we were ready to deploy to production we would follow a flow like this:

1. Open a PR to update the version based on the collective impact of the unreleased functionality.  This would also involve creating a new version header in the changelog.
2. Tag the commit that updates the version, using the appropriate version tag
3. perform the deployment

This is a little different from the continuous deployment philosophy we talked about last week, but it does allow us to batch the changelog.  Maybe the way to accomplish both is to have a "version number" feature flag that gets baked into our code base.

Anyway, this is all out of scope for this PR I think; I'll just strike this line and then we can talk more about versions / changelogs later!